### ISSUE TITLE
Please pin 2.0 RC dependencies

### ISSUE THREAD

**Issue by mikebm** [apollographql/apollo-server#1359]
This is more of a request then anything, but the RC versions of 2.0 have had a few breaking changes and even though I am pinning my versions, the packages themselves are using ^ versions which means when RC 7/8 was pubished, my version of RC6 started using dependencies of RC7/8. This is causing us to have dev down situations until we can update.  So my request is for the apollo team to start pinning their dependecies on the RC packages.

Thanks.

**Comment by evans** [comment]
Ah thank you for the issue! Sorry about that. We've pinned the dependencies with #1346. What sort of breaking changes did you experience? I'd like to have that sorted out as well.

**Comment by mikebm** [comment]
Awesome, thank you.

The first was coming off of RC0 to RC6. If I recall correctly, BadUserInputError was renamed to UserInputError. I believe RESTDataSource changed here as well, but don't recall if that was a breaking change.

Today we ran into the issue while on RC6. The dataSources function for GraphQLOptions was changed. This required RESTDataSource to extend from DataSource. I was forced to upgrade to RC8 today which also required changing how I start Apollo Server using Express.  This also affected the playground url which took a moment to figure out it is using the same route as the graphql queries now.
